extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -78,7 +78,7 @@ class MembershipsButtonEdit extends Component {
 			editedProductPriceValid: true,
 			editedProductTitle: '',
 			editedProductTitleValid: true,
-			editedProductRenewInterval: '1 month',
+			editedProductRenewInterval: 'one-time',
```

Comment:
I think we want to keep the default to monthly, based on https://github.com/Automattic/wp-calypso/pull/42848#issuecomment-639594153